apps/zero-runtime-next-app/next.config.js

```diff
@@ -3,7 +3,7 @@
 const { withZeroPlugin } = require('@mui/zero-next-plugin');
 const { experimental_extendTheme: extendTheme } = require('@mui/material/styles');
 
-const theme = extendTheme();
+const theme = extendTheme({ cssVarPrefix: 'app' });
```

Comment:
+1
But we'll have to do something similar for `createTheme`.

Reply:
Should we use this opportunity to force developers to transition to `extendTheme`? I lean toward having one API to create themes.

Some context about `createTheme` and `extendTheme`:

`createTheme` is the first API we have. To get dark mode, you will need to call it with `{ palette: { mode: 'dark' } }`. When we wanted to introduce the CSS variables, we needed to prepare all the palettes for both modes and that's why we introduced `extendTheme`.

With the trend and evolution of CSS, I think CSS vars will be the standard so that's why we should drop the `createTheme` at some point and teach people to use `extendTheme`.

Reply:
Actually, yeah. We can just make that the only approach to create theme while setting up the config. Anyways, users will be setting up build-time config for the first time. So we can document that using `extendTheme` is the only way to set it up.